#ENUM(Key: rightFootRight, rightFootLeft, leftFootRight, leftFootLeft, chest);


bool[5] pressed
